UnityProject/Assets/Scripts/Items/Traits/CommonTraits.cs

```diff
@@ -34,4 +34,5 @@ public class CommonTraits : SingletonScriptableObject<CommonTraits>
 	public ItemTrait Welder;
 	public ItemTrait Shovel;
 	public ItemTrait Knife;
+	public ItemTrait AtmosAnalyser;
```

Comment:
Should it really be a common trait? I mean, I can imagine 10 types of knives and why this is a trait but can't think of the same for atmos analyzer

Reply:
well how else do you want me to do it?
the whitelist system for the belt uses itemtraits

Reply:
Maybe we should instead have "Tool" trait which will allow items to be put in toolbelts/toolboxes and possibly something else

Reply:
the other reason i did it this way because the health analyzer was done the same way